ansible/roles/iroha-docker/defaults/main.yml

```diff
@@ -132,6 +132,9 @@ iroha_peer_keys_ansible_encrypt_enable: false
 # If you activated previous option you have to set path to your ansible vault file
 iroha_peer_keys_ansible_encrypt_password_file: ''
 
+# Enable it to use postgres instead of files for storing blocks (iroha 1.2.0+)
+iroha_enable_postgres_block_store: false
```

Comment:
Maybe just make `iroha_blockstore_path` parameter optional?

Reply:
Yes, you are right it would be better. I modified it